auditlog/models/rule.py

```diff
@@ -507,20 +547,25 @@ def create_logs(
         model_id = self.pool._auditlog_model_cache[res_model]
         auditlog_rule = self.env["auditlog.rule"].search([("model_id", "=", model_id)])
         fields_to_exclude = auditlog_rule.fields_to_exclude_ids.mapped("name")
+
+        vals = {
+            "model_id": model_id,
+            "method": method,
+            "user_id": uid,
+            "http_request_id": http_request_model.current_http_request(),
+            "http_session_id": http_session_model.current_http_session(),
+        }
+        vals.update(additional_log_values or {})
+        if method == "export_data":
+            vals.update({"res_ids": str(res_ids)})
```

Comment:
```suggestion
            vals.update({"name": res_model, "res_ids": str(res_ids)})
```
Just to assign something as a name, or the log record shows as 'New' when opened in a form. A trivial point.
![image](https://github.com/OCA/server-tools/assets/7766939/80e87770-c72f-4600-86b1-3cea2e37f3bb)

Reply:
Good idea 👍